docs/decisions/0012-initial-file-format.md

```diff
@@ -0,0 +1,86 @@
+# 12. Initial Supported File Format
+
+Date: 2022-06-30
+
+## Status
+
+Accepted
+
+## Context and Problem Statement
+
+For data storage, the team required an initial data format to support as the solution is developed. The file format will be used by other agencies to collect and analyze data health data in their region. The file format chosen will not be the only format supported by the application; in future iterations the team will support a variety of formats. However for the initial development of the application, only a few file formats will be supported 
+
+## Decision Drivers
+
+**Well-supported** - The file format must be common and well-supported by many applications. The users who may eventually use our files will use their own tools to read our data files. So having a common format will help end users access the data. 
+
+**Quick-to-implement** - Having a file format that can be easy to work with will help the development of the solution be quicker. 
+
+**Infrastructure requirements** - Due to limited resources on the team, a solution that requires less infrastructure resources to maintain will help speed up the development process.
+
+**Cost-effective** - With any cloud environment, storage cost considerations are necessary to scale up an application without incurring too much cost. 
+
+## Considered Options
+
+### Parquet
+Apache parquet is a free and open source storage binary format for fast analytical querying. Because of the binary storage format, it is easier to compress the file and therefore, less costly to store. It is a columnar oriented storage format instead of row format with the format being:
```

Comment:
To make it more clear what the first digram depicts, consider changing the last sentence here to something like:
`Rather than a row-oriented storage format as is common with CSV and RDBMS systems, Parquet uses a columnar data format such as:`

Reply:
added